--- libsmbios-2.2.26/src/python/Makefile.am.orig	2010-06-24 21:48:29.000000000 +0200
+++ libsmbios-2.2.26/src/python/Makefile.am	2010-08-14 16:23:59.846823295 +0200
@@ -32,5 +32,5 @@
 	cp $< $@
 	for var in $(REPLACE_VARS)	;\
 	do	\
-		perl -p -i -e "s|^$$var\s*=.*|$$var=\"$${!var}\"|"  $@;\
+		perl -p -i -e "s|^$$var\s*=.*|$$var=\"$$(eval echo \$$$$var)\"|"  $@;\
 	done
--- libsmbios-2.2.26/Makefile-std.orig	2009-06-23 21:08:54.000000000 +0200
+++ libsmbios-2.2.26/Makefile-std	2010-08-14 16:23:26.766824134 +0200
@@ -45,7 +45,7 @@ install-data-hook:
 	    file=$(DESTDIR)/$$i                     ;\
 	    for var in $(REPLACE_VARS)	;\
 	    do	\
-	        perl -p -i -e "s|^$$var\s*=.*|$$var=\"$${!var}\"|"  $$file;\
+	        perl -p -i -e "s|^$$var\s*=.*|$$var=\"$$(eval echo \$$$$var)\"|"  $$file;\
 	    done	;\
 	done
 
@@ -55,7 +55,7 @@ install-exec-hook:
 	    file=$(DESTDIR)/$$i                     ;\
 	    for var in $(REPLACE_VARS)	;\
 	    do	\
-	        perl -p -i -e "s|^$$var\s*=.*|$$var=\"$${!var}\"|"  $$file;\
+	        perl -p -i -e "s|^$$var\s*=.*|$$var=\"$$(eval echo \$$$$var)\"|"  $$file;\
 	    done	;\
 	done
 
