--- libsmbios-2.2.28/src/include/smbios/config/compiler/gcc.hpp.orig	2009-03-24 08:21:54.000000000 +0100
+++ libsmbios-2.2.28/src/include/smbios/config/compiler/gcc.hpp	2011-07-01 16:53:55.233257833 +0200
@@ -113,7 +113,7 @@
 #  error "Compiler not configured - please reconfigure"
 #endif
 //
-#if (__GNUC__ > 4) || ((__GNUC__ == 4) && (__GNUC_MINOR__ > 5))
+#if (__GNUC__ > 4) || ((__GNUC__ == 4) && (__GNUC_MINOR__ > 6))
 #  if defined(LIBSMBIOS_ASSERT_CONFIG)
 #     error "Unknown compiler version - please run the configure tests and report the results"
 #  else
