--- libsmbios-2.3.2/src/include/smbios/config/compiler/gcc.hpp.orig	2016-08-23 20:47:22.000000000 +0200
+++ libsmbios-2.3.2/src/include/smbios/config/compiler/gcc.hpp	2017-03-05 07:28:34.080882550 +0100
@@ -113,7 +113,7 @@
 #  error "Compiler too old. GCC > 3.0 required"
 #endif
 //
-#if (__GNUC__ > 5)
+#if (__GNUC__ > 6)
 #  if defined(LIBSMBIOS_ASSERT_CONFIG)
 #     error "Unknown compiler version - please run the configure tests and report the results"
 #  else
