diff -urNp -x '*.orig' libsmbios-2.4.3.org/Makefile-std libsmbios-2.4.3/Makefile-std
--- libsmbios-2.4.3.org/Makefile-std	2020-02-01 02:40:02.000000000 +0100
+++ libsmbios-2.4.3/Makefile-std	2021-03-05 22:52:24.235730489 +0100
@@ -38,7 +38,7 @@ install-data-hook:
 	    file=$(DESTDIR)/$$i                     ;\
 	    for var in $(REPLACE_VARS)	;\
 	    do	\
-	        perl -p -i -e "s|^$$var\s*=.*|$$var=\"$${!var}\"|"  $$file;\
+	        perl -p -i -e "s|^$$var\s*=.*|$$var=\"$$(eval echo \$$$$var)\"|"  $$file;\
 	    done	;\
 	done
 
@@ -48,7 +48,7 @@ install-exec-hook:
 	    file=$(DESTDIR)/$$i                     ;\
 	    for var in $(REPLACE_VARS)	;\
 	    do	\
-	        perl -p -i -e "s|^$$var\s*=.*|$$var=\"$${!var}\"|"  $$file;\
+	        perl -p -i -e "s|^$$var\s*=.*|$$var=\"$$(eval echo \$$$$var)\"|"  $$file;\
 	    done	;\
 	done
 
diff -urNp -x '*.orig' libsmbios-2.4.3.org/src/python/Makefile.am libsmbios-2.4.3/src/python/Makefile.am
--- libsmbios-2.4.3.org/src/python/Makefile.am	2020-02-01 02:40:02.000000000 +0100
+++ libsmbios-2.4.3/src/python/Makefile.am	2021-03-05 22:52:24.235730489 +0100
@@ -29,5 +29,5 @@ src/python/_vars.py: src/python/libsmbio
 	cp $< $@
 	for var in $(REPLACE_VARS)	;\
 	do	\
-		perl -p -i -e "s|^$$var\s*=.*|$$var=\"$${!var}\"|"  $@;\
+		perl -p -i -e "s|^$$var\s*=.*|$$var=\"$$(eval echo \$$$$var)\"|"  $@;\
 	done
